src/main.rs

```diff
@@ -102,6 +105,15 @@ fn settings() -> io::Result<Settings> {
                     DEFAULT_THRESHOLD.code_str(),
                 )),
         )
+        .arg(
+            Arg::with_name(KEEP_OPTION)
+                .value_name("REGEX")
+                .short("k")
+                .long(KEEP_OPTION)
+                .multiple(true)
+                .number_of_values(1)
+                .help("Prevents Docuum from deleting repository:tag images that match the provided <REGEX>"),
```

Comment:
Seems like the linter is complaining that this line is too long. Can we break up the string somewhere?